deepometry/commands/command_evaluate.py

```diff
@@ -90,19 +110,26 @@ def _evaluate(x, y, batch_size, directory, name, verbose):
     return model.model.metrics_names, metrics
 
 
-def _load(pathnames, labels):
+def _load(pathnames, labels, exclusion):
+
+    print('Before exclusion: ',len(pathnames))
+    pathnames = [x for x in pathnames if exclusion not in x]
+    print('After exclusion: ',len(pathnames))
+
     x = numpy.empty((len(pathnames),) + _shape(pathnames[0]), dtype=numpy.uint8)
 
     y = numpy.empty((len(pathnames),), dtype=numpy.uint8)
 
     label_to_index = {label: index for index, label in enumerate(sorted(labels))}
 
     for index, pathname in enumerate(pathnames):
-        label = os.path.split(os.path.dirname(pathname))[-1]
+        if os.path.isfile(pathname) == True: # in case there is a mixture of directories and files
```

Comment:
The `== True` in `if os.path.isfile(pathname) == True` is redundant. `if os.path.isfile(pathname)` is enough.